scripts/import/import_clinvar_xml

```diff
@@ -1093,9 +1128,17 @@ sub get_variant{
       print "Not entering new refSNP: $rs_id as no parsable HGVS available for alleles ($record->{feature_info}->{hgvs_g})\n" if $DEBUG == 1;
       return undef;
     }
-
+    
+     my ($new_var_ob, $var_feat);
     ## enter new records
-    my ($new_var_ob, $var_feat) = enter_var($record,  $ref_allele, $alt_allele, $rs_id, $record->{inheritance_type});
+    # enter the new record based on what we have at the moment 
+    ($new_var_ob, $var_feat) = enter_var($record,  $ref_allele, $alt_allele, $rs_id, $record->{inheritance_type}) if $record->{feature_info}->{hgvs_g} !~ /\=/;
+    # when it is a PAR region, we need to map based on the PAR region to the X or the Y 
+    if (defined $record->{feature_info}->{Chr_other}) {
+      $record->{feature_info}->{Chr} = ($record->{feature_info}->{Chr} eq "X") ? "Y" : "X";
+      # adding the mapped based on the PAR region
+      ($new_var_ob, $var_feat) = enter_var($record,  $ref_allele, $alt_allele, $rs_id, $record->{inheritance_type}) if $record->{feature_info}->{hgvs_g} !~ /\=/;
```

Comment:
The second time it calls the `enter_var`, `$var_feat` is going to be replaced. In the end, the method only returns one variation feature instead of two (one for chrX and the other for chrY). This is why we have evidence for only one variation feature.

Reply:
Update to:
```
my ($new_var_ob_2, $var_feat_2) = enter_var($record,  $ref_allele, $alt_allele, $rs_id, $record->{inheritance_type}) if $record->{feature_info}->{hgvs_g} !~ /\=/;
push @{$var_feat}, $var_feat_2->[0];
$new_var_ob = $new_var_ob_2;
```
This way you add another variation feature to `$var_feat`